### PR TITLE
QA8-1564

### PR DESCRIPTION
**Pull request by Slyvochka**

### CODE REVIEW

File: test/extended/story.js [hunk 1]
```diff
@@ -74,4 +74,32 @@ describe('Story field', function () {
...
+        inputValues4(name.default, gender.she, age.default, story.comedy);
+        $(sel.submit).click();
+        const capitalLetters =  countOfCapitalLet($$(sel.story)[0].getText());
+        expect(capitalLetters).toEqual(exp.totalCapitalLet);
+    });
+
+    it('TC-193 The story includes the moral', function () {
+        inputValues4(name.default, gender.she, age.default, story.comedy);
+        $(sel.submit).click();
+        const moral = $$(sel.story)[1];
```
Comment (isenkovska) [inline test/extended/story.js:102]: TC-193 failes, I think you need to add  getText(); to const moral;

**Review by isenkovska — CHANGES_REQUESTED** [review]
why are you girls in the same branch?) Iryna's tests look good. Story tests by Sveta need some improvements, I added comments

File: test/extended/story.js [hunk 2]
```diff
@@ -74,4 +74,32 @@ describe('Story field', function () {
         expect(isDisplayed).toEqual(true);
     });
 
+    it("TC-190 The story's title = 'My Little Hero'", function () {
+        inputValues4(name.default, gender.she, age.default, story.comedy);
+        $(sel.submit).click();
```
[the remark this answers is not in the document]
Comment (Slyvochka) [reply test/extended/story.js:79]: In which TC you meant

File: test/extended/story.js (hunk 1 above)
Comment (Slyvochka) [reply test/extended/story.js]: corrected

File: test/extended/story.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (isenkovska) [reply test/extended/story.js:79]: all of them TC-190 -TC-193

File: data/selectors.js [hunk 3]
```diff
@@ -31,6 +31,7 @@ const elements = {
     "submit": ".ant-btn",
     "imageStoryPage": "img[alt='hero_picture']",
     "story": ".card-text",
+    "storyName": "h4",
```
[the remark this answers is not in the document]
Comment (Slyvochka) [reply data/selectors.js:34]: usually title level tag from h1 to h6 unique, so why can't I use it

File: test/extended/story.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (isenkovska) [reply test/extended/story.js:79]: sorry, just checked the method, just in TC-192 this method can be used

**Review by isenkovska — APPROVED** [review]

**Review by EAGauss — CHANGES_REQUESTED** [review]
You have worked with the old version of the main branch so that you had not included the changes from our code review in your branch. I will merge them to your branch so as not to lose our progress.

**Review by EAGauss — APPROVED** [review]
Looks good now!

**Pull request merged**